src/Func.jl

```diff
@@ -13,7 +13,7 @@ const OR = Func(|, 2)
 const XOR = Func($, 2)
 const NOT = Func(~, 1)
 const ZERO = Func(() -> convert(BitString, 0), 0, "0")
-const ONE = Func(() -> convert(BitString, ~0), 0, "1")
+const ONE = Func(() -> ~convert(BitString, 0), 0, "1")
```

Comment:
I can work on adding tests of goal composition and fitness to
test/Goals.jl.  It may be tomorrow before I get anything done.  Do you
want to delay merging goals.cleanup until I get this done?

Alden

On Wed, Mar 11, 2015, at 12:52, George Lesica wrote:
> > @@ -13,7 +13,7 @@ const OR = Func(|, 2)
> >  const XOR = Func($, 2)
> >  const NOT = Func(~, 1)
> >  const ZERO = Func(() -> convert(BitString, 0), 0, "0")
> > -const ONE = Func(() -> convert(BitString, ~0), 0, "1")
> > +const ONE = Func(() -> ~convert(BitString, 0), 0, "1")
> 
> This was done because in Julia 0.4 the `~0` syntax resulted in a loss of
> precision error.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/glesica/CGP.jl/pull/4/files#r26243394